Mathlib/SetTheory/Ordinal/FixedPoint.lean

```diff
@@ -60,7 +60,8 @@ theorem foldr_le_nfpFamily (f : ι → Ordinal → Ordinal)
   Ordinal.le_iSup _ _
 
 theorem le_nfpFamily (f : ι → Ordinal → Ordinal) (a) : a ≤ nfpFamily f a :=
-  Ordinal.le_iSup _ []
+  Ordinal.le_iSup (fun _ ↦ List.foldr _ a _) []
+
```

Comment:
```suggestion
```

Reply:
I'm just gonna go ahead and apply this suggestion, since it's pretty uncontroversial and I kinda need this PR to go through for some of my own.